gpu/kinfu/include/pcl/gpu/kinfu/kinfu.h

```diff
@@ -203,7 +204,7 @@ namespace pcl
       private:
         
         /** \brief Number of pyramid levels */
-        enum { LEVELS = 3 };
+        static constexpr size_t LEVELS = 3;
```

Comment:
```suggestion
        static constexpr std::size_t LEVELS = 3;
```

Reply:
Obsolete, as I removed my last commit, because `static constexpr` is a [C++17 feature](https://en.cppreference.com/w/cpp/language/definition) and not C++14 :-(